src/pages/en/guides/troubleshooting.md

```diff
@@ -11,16 +11,17 @@ Astro provides several different tools to help you troubleshoot and debug your c
 
 Here are some common error messages you might see in the terminal, what they might mean, and what to do about them.
 
-### Transform failed with X error
+### Transform failed with "Unexpected `export`"
 
-This message often appears due to a current limitation in Astro requiring your import and export statements to be at the top of your `.astro` file.
+A current limitation in Astro is that `export` statements (other than `getStaticPaths`) are not supported in `.astro` files.
```

Comment:
Can’t we also `export interface Props`?

Reply:
```suggestion
A current limitation in Astro is that `export` statements (other than `getStaticPaths()`) are not supported in `.astro` files.
```
I noticed that we are not *entirely* consistent in how we refer to `getStaticPaths()` in regular sentence text. I know I do typically include the parenthesis, but this is not 100% standard throughout the Docs.

Can we pick one way, whichever way, and whatever we use here, I'll make sure it's then reflected throughout the site?